main.js

```diff
@@ -1,82 +1,87 @@
+    }
+    return books;
+  }
 
-  if (books.length === 0) {
-    id = 1;
-  } else {
-    id = [(books.length - 1) + suma];
+  static addBookToStorage(book) {
+    const books = LocalStorage.loadBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
   }
-  const t = document.getElementById('title').value;
-  const a = document.getElementById('author').value;
-  const y = new Book(t, a);
-  books.push(y);
-  const bookData = `
-    <div id="${id}">
-        <h2>${y.title}</h2>
-        <p>${y.author}</p>
-        <button id="${y}" class="remove">x</button>
-        <hr>
-    </div>
-    `;
-  book.innerHTML += bookData;
-  const titleValue = document.getElementById('title');
-  const authorValue = document.getElementById('author');
-  titleValue.value = '';
-  authorValue.value = '';
-  localStorage.setItem('books', JSON.stringify(books));
-});
 
-forma.addEventListener('submit', (e) => {
-  e.preventDefault();
-  addSingleBook();
-});
+  static removeBookFromStorage(title) {
+    const books = LocalStorage.loadBooks();
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
 
-const removeSingleBook = (e) => {
-  if (e.target.textContent === 'x') {
-    const booksId = e.target.parentElement;
-    const { id } = e.target;
-    books.splice(id, 1);
-    booksId.remove();
+class Interface {
+  static booksDisplay() {
+    const books = LocalStorage.loadBooks();
+    books.forEach((book) => Interface.addBook(book));
   }
-  localStorage.setItem('books', JSON.stringify(books));
-};
 
-book.addEventListener('click', (e) => {
-  removeSingleBook(e);
-});
+  static addBook(book) {
+    const list = document.getElementById('book-list');
+    const row = document.createElement('tr');
+    row.innerHTML = `
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td></td>
+      <td><a href="#" class="btn btn-danger btn-sm delete">X</a></td>  
+      `;
+    list.appendChild(row);
+  }
 
-window.addEventListener('load', () => {
-  const content = JSON.parse(localStorage.getItem('books'));
+  static removeBook(target) {
+    if (target.classList.contains('delete')) {
+      target.parentElement.parentElement.remove();
+    }
+  }
 
-  if (content === 0 || content === null) {
-    localStorage.setItem(('books', JSON.stringify(books)));
-  } else {
-    books = content;
-    addBook();
+  static clearValues() {
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
   }
+}
+
+document.addEventListener('DOMContentLoaded', Interface.booksDisplay);
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const book = new Book(title, author);
+  Interface.addBook(book);
```

Comment:
[Optional] Currently, I am able to save entries even when they're only made of spaces or when the fields are empty. I suggest that you trim the input values before saving in order to make sure that the user entered non-space characters.

![image](https://user-images.githubusercontent.com/20554583/204893116-00b82135-2dab-4787-a11c-e254184ca44f.png)